packages/camera/camera_web/lib/src/camera.dart

```diff
@@ -41,11 +41,12 @@ class Camera {
   /// Creates a new instance of [Camera]
   /// with the given [textureId] and optional
   /// [options] and [window].
-  Camera({
-    required this.textureId,
-    required CameraService cameraService,
-    this.options = const CameraOptions(),
-  }) : _cameraService = cameraService;
+  Camera(
+      {required this.textureId,
+      required CameraService cameraService,
+      this.options = const CameraOptions(),
+      this.recorderOptions = const (audioBitrate: null, videoBitrate: null)})
```

Comment:
```suggestion
      this.recorderOptions = const (audioBitrate: null, videoBitrate: null),})
```

(Adding a comma after `this.recorderOptions = ...` might make the formatting of the Camera constructor signature a little bit nicer!)